superset/connectors/sqla/views.py

```diff
@@ -167,9 +170,7 @@ class TableModelView(SupersetModelView, DeleteMixin):  # noqa
         'schema': _(
             "Schema, as used only in some databases like Postgres, Redshift "
             "and DB2"),
-        'description': Markup(
-            "Supports <a href='https://daringfireball.net/projects/markdown/'>"
-            "markdown</a>"),
+        'description': _("Supports markdown"),
```

Comment:
As the comment of `lazy_gettext` state, it will be translated when it is used as an actual string.

But methods `Markup()` and `utils.markdown()` are called when the server is loading, Which means the `LazyString` is used,  and the `LazyString` is converted to string when `_get_current_context()` is None.

```python
def get_translations():
    """Returns the correct gettext translations that should be used for
    this request.  This will never fail and return a dummy translation
    object if used outside of the request or if a translation cannot be
    found.
    """
    ctx = _get_current_context()

    if ctx is None:
        return support.NullTranslations()

    # ...
```

```python
if PY2:
    text_type = unicode
    string_types = (str, unicode)
else:
    text_type = str
    string_types = (str, )

class LazyString(object):
    # ...
    def __str__(self):
        return text_type(self._func(*self._args, **self._kwargs))
```